admin-auth/src/test/kotlin/com/epam/drill/admin/auth/CacheServiceImplTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.epam.drill.admin.auth
+
+import com.epam.drill.admin.auth.principal.Role
+import com.epam.drill.admin.auth.principal.User
+import com.epam.drill.admin.auth.service.impl.CacheServiceImpl
+import com.github.benmanes.caffeine.cache.Caffeine
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert
+import java.time.Duration
+import kotlin.test.Test
+
+/**
+ * Tests for [CacheServiceImpl] logic
+ */
+class CacheServiceImplTest {
+
+    @Test
+    fun `given two distinct apiKeys, then cache size should be two`(): Unit = runBlocking {
```

Comment:
It would be better to rework this test taking into account the comment for the test below

Reply:
Refactored. The changes are open for discussion and review.